pallets/did/src/mock_utils.rs

```diff
@@ -93,12 +93,16 @@ pub fn generate_base_did_creation_details<T: Config>(
 	}
 }
 
-pub fn generate_base_did_details<T: Config>(authentication_key: DidVerificationKey) -> DidDetails<T> {
+pub fn generate_base_did_details<T>(authentication_key: DidVerificationKey) -> DidDetails<T>
+where
+	T: crate::Config,
```

Comment:
We already import `Config`, so we can use the same.

Reply:
825a7fb